src/components/AccountDetailsCard.js

```diff
@@ -20,7 +20,7 @@ import PropTypes from 'prop-types';
 import React from 'react';
 import { StyleSheet, Text, View } from 'react-native';
 import colors from '../colors';
-import { DEFAULT_NETWORK_COLOR, NETWORK_COLOR, NETWORK_TITLES } from '../constants';
+import { NETWORK_LIST } from '../constants';
 import AccountIcon from './AccountIcon';
 import TouchableItem from './TouchableItem';
 
```

Comment:
This class isn't used anywhere, we can remove the whole file I guess.

Reply:
True, probably we should set up the `lint` and `prettier` in `pre-commit`

Reply:
This one is still in use.